mms2/java-app-ps/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine
+RUN apk update;apk add shadow bash openjdk8 apache-ant wget unzip
+WORKDIR /opt/code/libs
+RUN wget -q https://oss.sonatype.org/content/repositories/snapshots/io/netty/netty-all/4.0.56.Final-SNAPSHOT/netty-all-4.0.56.Final-20180202.121251-12.jar
+RUN wget -q http://downloads.datastax.com/java-driver/cassandra-java-driver-3.5.0.tar.gz;tar xf cassandra-java-driver-3.5.0.tar.gz
```

Comment:
I remember we plan to use Scylla driver
https://github.com/scylladb/java-driver/releases/tag/3.6.0

Reply:
from here https://github.com/scylladb/java-driver/releases

Reply:
Originally we did but then we found out that it doesn't compile (I believe you opened an issue about it?), and decided to use the Datastax driver